templates/base/packages/nextjs/.env.example.template.mjs

```diff
@@ -1,13 +1,23 @@
-# Template for NextJS environment variables.
+import { withDefaults } from "../../../utils.js";
+
+const contents = ({ additionalVars }) => {
+  return `# Template for NextJS environment variables.
 
 # For local development, copy this file, rename it to .env.local, and fill in the values.
 # When deploying live, you'll need to store the vars in Vercel/System config.
 
-# If not set, we provide default values (check `scaffold.config.ts`) so developers can start prototyping out of the box,
+# If not set, we provide default values (check scaffold.config.ts) so developers can start prototyping out of the box,
```

Comment:
```suggestion
# If not set, we provide default values (check \`scaffold.config.ts\`) so developers can start prototyping out of the box,
``` 

maybe add this, just to keep things same as previous?

Reply:
[12c213a](https://github.com/scaffold-eth/create-eth/pull/74/commits/12c213a437296a52f4db1d03153809ab8b4c2811) Thanks!